src/Features/Core/Portable/EditAndContinue/Utilities/Extensions.cs

```diff
@@ -183,9 +183,38 @@ private static bool ParsePrimaryParameterBackingFieldName(string fieldName, [Not
         => (IMethodSymbol?)constructor.ContainingType.GetMembers(WellKnownMemberNames.DeconstructMethodName).FirstOrDefault(
             static (symbol, constructor) => symbol is IMethodSymbol method && HasDeconstructorSignature(method, constructor), constructor)?.PartialAsImplementation();
 
-    // https://github.com/dotnet/roslyn/issues/73772: does this helper need to be updated to use IPropertySymbol.PartialImplementationPart?
+    /// <summary>
+    /// Returns a partial implementation part of a partial member, or the member itself if it's not partial.
+    /// </summary>
     public static ISymbol PartialAsImplementation(this ISymbol symbol)
-        => symbol is IMethodSymbol { PartialImplementationPart: { } impl } ? impl : symbol;
+        => symbol switch
+        {
+            IMethodSymbol { PartialImplementationPart: { } impl } => impl,
+            IPropertySymbol { PartialImplementationPart: { } impl } => impl,
+            _ => symbol
+        };
```

Comment:
Nit: This seems to be just `=> PartialImplementationPart(symbol) ?? symbol`. Need both?